README.md

```diff
@@ -72,8 +72,13 @@ EOF
 To create an example cluster:
 
 ```shell
-clusterctl generate cluster docker-quick-start-helm-addon-cilium \
-  --from examples/capi-quick-start/docker-cluster-cilium-helm-addon.yaml \
+export CLUSTER_NAME=docker-cluster-cilium-helm-addon
+export CLUSTER_FILE=examples/capi-quick-start/docker-cluster-cilium-helm-addon.yaml
```

Comment:
I haven't removed the env yet because I still think its easier to set it at the top to make the command more copy and pastable. 

(My IDE even gives me a button to run the commands)
<img width="505" alt="image" src="https://github.com/d2iq-labs/capi-runtime-extensions/assets/4969231/6d57817d-9192-49ae-9f25-71de2cb8aae9">